uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
If you're modifying this anyway, can you push this declaration to be below the columns?  That's where we usually put our ``relationship`` declarations elsewhere in ``models.py``.